src/components/blog/BlogList.js

```diff
@@ -111,76 +114,88 @@ class BlogList extends Component {
-                                                         alt={blogItem.title}
-                                                    />
+                                <Link to={`/blogitem/${blogItem.slug}`}
+                                      style={{textDecoration: 'none', color: 'black'}}>
+                                    <div className="m-4 rounded card card-body clickable"
+                                         onClick={this.showBlogItem(blogItem.slug)}>
+
+                                        <div className="row">
+                                            <div className="col-md-5">
+                                                <div className="post-thumbnail">
+
+                                                    {blogItem.image ?
+                                                        <img className="card-img-top"
+                                                             height="250"
+                                                             src={blogItem.image}
+                                                             alt={blogItem.title}
+                                                        />
+                                                        :
+                                                        <img className="card-img-top"
+                                                             height="250"
+                                                             src={defaultImage}
+                                                             alt={blogItem.title}
+                                                        />
+                                                    }
+
+                                                </div>
+                                            </div>
+                                            <div className="col-md-7">
+                                                <header className="entry-header">
+                                                    <h3 className="entry-title">{blogItem.title}</h3>
+                                                    <ul className="entry-meta list-inline">
+                                                        <li> Posted by
+                                                            <span className="posted-by">
+                                                                <span className="author"> {blogItem.owner}</span>
+                                                            </span>
+                                                        </li>
+                                                        <li>Posted on
+                                                            <span className="posted-on">
+                                                                <time className="updated"
+                                                                      dateTime="2019-07-26T10:33:42+00:00"> {(new Date(blogItem.created)).toDateString()}
```

Comment:
nit: remove this hard coded `dateTime="2019-07-26T10:33:42+00:00"`

Reply:
Done.